packages/firebase_remote_config/firebase_remote_config_platform_interface/lib/src/method_channel/method_channel_firebase_remote_config.dart

```diff
@@ -63,8 +60,8 @@ class MethodChannelFirebaseRemoteConfig extends FirebaseRemoteConfigPlatform {
 
   @override
   FirebaseRemoteConfigPlatform setInitialValues(
-      {Map<dynamic, dynamic> remoteConfigValues}) {
-    final fetchTimeout = Duration(seconds: remoteConfigValues['fetchTimeout']);
+      {Map<dynamic, dynamic>? remoteConfigValues}) {
+    final fetchTimeout = Duration(seconds: remoteConfigValues!['fetchTimeout']);
```

Comment:
It doesn't make sense to use ! if the parameter is optional.

Make it required instead